apis/python/src/tiledbsoma/sparse_nd_array.py

```diff
@@ -26,7 +25,7 @@
 _UNBATCHED = options.BatchSize()
 
 
-class SparseNDArray(TileDBArray, somacore.SparseNDArray):
+class SparseNDArray(NDArray, somacore.SparseNDArray):
```

Comment:
Can the `create()` class method be removed from `SparseNDArray` as well?

Reply:
How did I manage to miss that? And the other things that are also shared in common? Feeling pretty silly about that right now.